lib/src/charts/widgets/core/cartesian_charts.dart

```diff
@@ -7,22 +6,26 @@ import 'package:flutter/material.dart';
 class CartesianCharts extends StatefulWidget {
   final double? width;
   final double? height;
-  final CartesianChartStyle style;
 
-  // Mandatory Fields
-  final CartesianPaintConstructor constructPainters;
+  final double? uMinXValue;
+  final double? uMaxXValue;
+  final double? uMinYValue;
+  final double? uMaxYValue;
```

Comment:
What are these values? Where do we need them?
I could not find any usages for this

Reply:
same, user defined min/max values. We won't use our calculated min/max values if these are defined